incubating/service-now/README.md

```diff
@@ -0,0 +1,68 @@
+# ServiceNow integration
+
```

Comment:
Consider adding a sentence here to describe what the step does, including the callback aspect.